topo/node/cisco/cisco.go

```diff
@@ -368,7 +368,18 @@ func defaults(pb *tpb.Node) (*tpb.Node, error) {
 		}
 	}
 	if pb.Config == nil {
-		pb.Config = &tpb.Config{}
+		pb.Config = &tpb.Config{
+			Image: "us-west1-docker.pkg.dev/gep-kne/cisco/xrd:ga",
+			Cert: &tpb.CertificateCfg{
```

Comment:
cisco does not support this block, they are configured according to the cfg of the device

however adding it would be more accurate when looking at the proto even though its a no-op technically. can you look on a device and see what certname and keyname get generated?

Reply:
Ack. Found names "ems.pem" and "ems.key" under /xr-storage/config/grpc for certname and keyname. Defaulted to these values.